Contributing.md

```diff
@@ -0,0 +1,57 @@
+# Branch Structure
+* **master**: The latest official GA version, which is currently 5.x
+* **dev**: The dev working branch of master, which is 5.x
+* **dev4x**: The dev working branch of master for 4.x
```

Comment:
dev4x isn't really a branch of master; it's really a branch of release4x, right?